docs-src/concepts.md

```diff
@@ -185,6 +186,15 @@ The API does not specify which one of these actions will be taken. Note that a
 GatewayClass controller that manages in-cluster proxy processes MAY restrict
 Gateway configuration scope, e.g. only be served in the same namespace.
 
+#### Gateway Status
+
+Gateways track status for the Gateway resource as a whole as well as each
+Listener it contains. Status can be tracked indepently on each Route resource.
+Within GatewayStatus, Listeners will have status entries corresponding to their
+name. Both GatewayStatus and ListenerStatus follow the conditions pattern used
+elsewhere in Kubernetes. This is a list that includes a type of condition, the
+status of that condition, and the last time this condition changed.
```

Comment:
"GatewayStatus", "Listener", and "ListenerStatus" need mark-up for consistency with the rest of the document: ``` `GatewayStatus` ```, ``` `Listener` ```, ``` `ListenerStatus` ```.

Suggested rewording for this sentence: "Namely, the `conditions` field of `GatewayStatus` or `ListenerStatus` has a list of conditions, where each condition has a unique type, a status, the last time the condition changed, and optionally a reason and message with more details about the condition."

Reply:
I think I got all of these, but there were some I wasn't quite sure on. It looks like we have some inconsistency throughout around where we use backticks or not.